frontends/web/src/routes/account/send/send.tsx

```diff
@@ -363,15 +354,6 @@ class Send extends Component<Props, State> {
     return Object.keys(this.selectedUTXOs).length !== 0;
   };
 
-  private toggleCoinControl = () => {
-    this.setState(({ activeCoinControl }) => {
-      if (activeCoinControl) {
-        this.selectedUTXOs = {};
```

Comment:
note for myself: I was quickly confused why this is gone, but it looks like toggleCoinControl was only used to open the dialog and activeCoinControl never true.